ramls/types/resources/resourcesRelationships.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Metadata of relationships for a given object",
+  "description": "Metadata of relationships for a given object",
+  "javaType": "org.folio.rest.jaxrs.model.ResourceRelationships",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "vendor": {
```

Comment:
we do not need "vendor" anymore. It is redundant with "provider". History is that earlier "vendor" in RM API was mapped to "vendor" in mod-kb but on usability testing request, we mapped "vendor" to "provider" and cleaned up "vendor" from mod-kb-ebsco but looks like its not cleaned completely. UI only recognizes "provider" and does not know "vendor". So, we can safely remove "vendor"

Reply:
the reason why "vendor" was added to the resourcesRelationships.json is that I have performed a request from the collection `../eholdings/providers/19?include=packages` and found following objects inside a relationship
![image](https://user-images.githubusercontent.com/37537790/47704419-3cb58180-dc2c-11e8-8ceb-d98d0e81079c.png)
@sduvvuri-ebsco correct me if I am wrong, we will have only two objects  "resources" and "provider", right?

Reply:
yes @natali-zaitseva , understand why you have it there in the first place and in mod-kb-ebsco-java, we can only return "resources" and "provider" and eliminate "vendor". Thanks!